extensions/script/tests/CMakeLists.txt

```diff
@@ -73,6 +80,17 @@ FOREACH(testfile ${EXECUTEPYTHONPROCESSOR_UNIT_TESTS})
     createTests("${testfilename}")
     add_test(NAME "${testfilename}" COMMAND "${testfilename}"  WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})
     MATH(EXPR EXTENSIONS_TEST_COUNT "${EXTENSIONS_TEST_COUNT}+1")
+
+    if(EXTENSIONS_TEST_COUNT EQUAL 1)
```

Comment:
same here

Reply:
Updated in f0fa8ce115fe1c892e25d14d87b6bbb26c586f1f